server/protocols/webadmin-cli/src/test/java/org/apache/james/cli/MailboxManageTest.java

```diff
@@ -185,4 +185,68 @@ void mailboxListWithNonExistingUsernameShouldFail(GuiceJamesServer server) {
         assertThat(outputStreamCaptor.toString()).isEmpty();
     }
 
+    @Test
+    void mailboxDeleteAParentMailboxWithTwoAddedChildrenMailboxShouldDeleteThemAll(GuiceJamesServer server) throws Exception {
+        Port port = server.getProbe(WebAdminGuiceProbe.class).getWebAdminPort();
+        dataProbe = server.getProbe(DataProbeImpl.class);
+        dataProbe.fluent().addDomain("linagora.com")
+            .addUser("hqtran@linagora.com", "123456");
+
+        WebAdminCli.executeFluent(new PrintStream(new ByteArrayOutputStream()), new PrintStream(new ByteArrayOutputStream()),
+            "--url", "http://127.0.0.1:" + port.getValue(), "mailbox", "create", "hqtran@linagora.com", "INBOX.1");
+
+        WebAdminCli.executeFluent(new PrintStream(new ByteArrayOutputStream()), new PrintStream(new ByteArrayOutputStream()),
+            "--url", "http://127.0.0.1:" + port.getValue(), "mailbox", "create", "hqtran@linagora.com", "INBOX.2");
+
+        int exitCode = WebAdminCli.executeFluent(new PrintStream(outputStreamCaptor), new PrintStream(errorStreamCaptor),
+            "--url", "http://127.0.0.1:" + port.getValue(), "mailbox", "delete", "hqtran@linagora.com", "INBOX");
+
+        WebAdminCli.executeFluent(new PrintStream(outputStreamCaptor), new PrintStream(errorStreamCaptor),
+                "--url", "http://127.0.0.1:" + port.getValue(), "mailbox", "list", "hqtran@linagora.com");
```

Comment:
what is this used for? I'm real confused on what your last assert is supposed to check... this message?

Reply:
Actually this and deleteAll test are the same, both of them assert the result of OutputStream of delete/deleteAll + list .
I fixed it to 
```
assertThat(outputStreamCaptor.toString()).isEqualTo("The mailbox now does not exist on the server.\n"
            + "");
``` to more readable.

Does this help you less confused?

Reply:
Shouldn't it be the contrary?
You are trying to list the mailboxes of a user here. 

`The mailbox now does not exist on the server` -> I would expect to see this message on the exist mailbox command when the mailbox does not exist, because you are looking for a particular mailbox.

In a list, you just want all mailboxes of a user, not a particular one. If there is none, I wouldexpect more something like `The user does not have mailboxes anymore` actually.

Reply:
Actually, ```The mailbox now does not exist on the server``` is the result of delete command, and list command in this case prints nothing.
In the case there is none mailbox, this is why i don't do something like ```The user does not have mailboxes anymore``` :
![image](https://user-images.githubusercontent.com/55171818/102166653-2aa40500-3ebf-11eb-9b96-5190930e8bac.png)

Reply:
Ok got it. add just a little comment in the test to explain that then, because it's still misleading :)